packages/next/auth.jsx

```diff
@@ -29,6 +29,7 @@ const userFragment = `
   phone
   email
   isAdmin
+  isSupport
```

Comment:
No there is an example. There is no effect on anything! don't change it

Reply:
Why not change? Flag "isSupport" is not sent from the server